utilities/openemr/openemr-cmd

```diff
@@ -0,0 +1,307 @@
+#!/bin/bash
+#########################################################
+# Function :Manage the docker and docker-compose cmd    #
+# Version  :v1                                          #
+#########################################################
+DOCKER_EXEC_CMD='docker exec -i'                                 # docker exec command
+CONTAINER_ID=$(docker ps | grep _openemr | cut -f 1 -d " ")      # get the container id
+SHELL_TYPE='sh -c'                                               # specify shell type
+
+refresh_page_or_iframe() {
+	$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE '/root/devtools build-themes'
+}
+
+check_php_error_logs() {
+	$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE '/root/devtools php-log'
+}
+
+create_psr12_code_styling_issues_report() {
+	$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE '/root/devtools psr12-report'
+}
+
+fix_psr12_code_styling_issues() {
+	 $DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE '/root/devtools psr12-fix'
+}
+
+create_theme_styling_issues_report() {
+	$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE '/root/devtools lint-themes-report'
+}
+
+fix_theme_styling_issues() {
+	$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE '/root/devtools lint-themes-fix'
+}
+
+check_php_parsing_errors() {
+	$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE '/root/devtools php-parserror'	
+}
+
+run_unit_testing() {
+	$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE '/root/devtools unit-test'
+}
+
+run_api_testing() {
+	$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE '/root/devtools api-test'
+}
+
+run_e2e_testing() {
+	$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE '/root/devtools e2e-test'
+}
+
+run_services_testing() {
+	$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE '/root/devtools services-test'
+}
+
+run_fixtures_testing() {
+	$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE '/root/devtools fixtures-test'
+}
+
+run_validators_testing() {
+	$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE '/root/devtools validators-test'
+}
+
+run_controllers_testing() {
+	$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE '/root/devtools controllers-test'
+}
+
+run_common_testing() {
+	$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE '/root/devtools common-test'
+}
+
+run_the_entire_dev_tool_suite() {
+	$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE '/root/devtools clean-sweep'
+}
+
+run_only_all_the_autmated_tests() {
+	$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE '/root/devtools clean-sweep-tests'
+}
+
+reset_openemr_only() {
+	$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE '/root/devtools dev-reset'
+}
+
+reset_and_reinstall_openemr() {
+	$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE '/root/devtools dev-reset-install'
+}
+
+reset_and_reinstall_openemr_with_demodata(){
+	$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE '/root/devtools dev-reset-install-demodata'
+}
+
+creat_a_backup_snapshot(){
+	$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE '/root/devtools backup example'
```

Comment:
oops, didn't notice this. this should also take a parameter rather than hard-code example